### PR TITLE
Let there be biases

### PR DESCRIPTION
**Pull request by jyegerlehner**
Seems to be working a bit better with these.

screenshot from 2016-09-18 20 41 17

The orange line is without biases, and blue line is same model with the biases.

### CODE REVIEW

**Comment by ibab** [comment]
This is interesting. There don't seem to be biases in the original paper.
Could you turn these into an optional parameter in the `wavenet_params.json` file?
As in `use_biases: true`.

**Comment by jyegerlehner** [comment]
> There don't seem to be biases in the original paper.

Well, to me it appears to be a detail they didn't go into. One of those "it goes without saying" things. Who ever builds a neural net without biases? It would be tedious to put that verbiage in the paper IMO.

> Could you turn these into an optional parameter in the wavenet_params.json file?

OK

**Comment by ibab** [comment]
> Well, to me it appears to be a detail they didn't go into. One of those "it goes without saying" things. Who ever builds a neural net without biases? It would be tedious to put that verbiage in the paper IMO.

That's actually a good point.
Thinking about this, at least the 1x1 layers should definitely have biases, and they should be beneficial for the causal convolutions as well.
I'm pretty convinced that I've interpreted the paper too literally in this regard.

**Comment by jyegerlehner** [comment]
I've re-run code where the biases are disabled and I get the same results as the current master branch. I think my loss plot above is wrong: the difference does not apear to be due to biases. I would guess due to some other change I pulled in to rebase on master. I'm closing this until I figure out what's going on.

**Comment by jyegerlehner** [comment]
Superceded by https://github.com/ibab/tensorflow-wavenet/pull/57
